go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/mattn/go-isatty
+module github.com/mysqto/isatty
```

Comment:
Please revert this.

Reply:
Sorry, I commit some personal code before this PR is merged